Makefile.defs

```diff
@@ -313,12 +313,15 @@ LIBTOOLIZE:=$(if $(DARWIN),glibtoolize,libtoolize)
 # (Also, it was a pain to deal with because of libtool,
 # c.f., https://www.gnu.org/software/libtool/manual/html_node/Stripped-link-flags.html#Stripped-link-flags).
 ifndef DARWIN
-	# TODO: Detect/support libc++ Clang TCs?
-	ifdef ANDROID
-		SHARED_STL_LIB:=$(shell PATH='$(PATH)' $(CC) -print-file-name=libc++_shared.so)
-	else
-		SHARED_STL_LIB:=$(shell PATH='$(PATH)' $(CC) -print-file-name=libstdc++.so.6)
-	endif
+  # TODO: Detect/support libc++ Clang TCs?
```

Comment:
I have some make nastiness for that, FWIW.

(Probably ripped from Linux's makefiles, IIRC).